modules/ipi-install-configuring-bios-for-worker-node.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id="configuring-bios-for-worker-node_{context}"]
+= Configuring BIOS for worker node
+
+During the installation process, the BIOS of the worker node can be configured
+by the following steps.
+
+.Procedure
+. Create manifests.
+. Modify the BMH file corresponding to the worker:
++
+----
+$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-3.yaml
+----
+. Add the BIOS configuration to the `spec` section of the BMH file:
++
+----
+spec:
+  firmware:
+    simultaneousMultithreadingEnabled: true
+    sriovEnabled: true
+    virtualizationEnabled: true
+----
++
+[NOTE]
+====
+. Only support these three BIOS configurations. See the
```

Comment:
```suggestion
Red Hat supports three BIOS configurations. See the link:https://github.com/openshift/baremetal-operator/blob/master/docs/api.md#firmware[BMH documentation] for details. Only servers with bcc type `irmc` are supported. Other types of servers are currently not supported. 
```

I would put this in 1 NOTE versus making it layered since it's the same type of content.

Reply:
> Other types of servers are currently not supported. 

From the code point of view, there are some other types of servers that support this feature, but we don't have a machine for testing. So I think it's better to explain that other types of services are not tested, rather than  not supported.

Reply:
updated, PTAL.

Reply:
thanks the update.
I think RH can't support not tested servers, so this modification is needed.